src/components/editArrangementTitle/editArrangementTitle.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { Typography, TextField } from '@material-ui/core'
+
+export class EditArrangementTitle extends Component {
+    constructor (props) {
+        super(props)
+
+        this.handleKeyPress = this.handleKeyPress.bind(this)
+        this.escFunction = this.escFunction.bind(this);
+    }
+    
+    state = {
+        name: ''
+      };
+   
+    handleChange = name => event => {
+        this.setState({
+          name: event.target.value
+        });
+      };
+    
+    handleKeyPress (event) {
+        if (event.key === 'Enter') {
+            this.setState({
+                name: this.state.name
+            });
+            this.props.handleEnter(this.state.name)
+        }
+    }
+
+    escFunction (event) {
+        if(event.keyCode === 27) {
```

Comment:
Nitpick: Can we change 27 into a "ESC_KEY" const?

Reply:
agreed, just define the constant at the top of the file.

Reply:
done